config/crd/bases/ptp.openshift.io_ptpconfigs.yaml

```diff
@@ -50,7 +50,6 @@ spec:
                     ptp4lOpts:
                       type: string
                   required:
-                  - interface
```

Comment:
you need this change also here

https://github.com/openshift/ptp-operator/tree/master/manifests/4.9

Reply:
done